packages/plugin-prisma-utils/tests/index.test.ts

```diff
@@ -24,7 +24,7 @@ describe('prisma utils', () => {
       query {
         posts(
           order: { author: { name: Desc, profile: null } }
-          filter: { id: { not: { equals: 11, not: null } } }
```

Comment:
Why did this need to be removed?

Reply:
because the `id` field not be null

Reply:
<img width="530" alt="image" src="https://github.com/hayes/pothos/assets/10432452/7b087ab8-38e4-45c8-a6e9-b2249c182da6">  

Prisma only allow use null filter on null field,  sush as the `content` field